pages/mfbarrera/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Hotels</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <section>
+    <div class="container">
+      <div class="background">
+        <div class="carousel__header">OUR HOTELS</div>
+        <div class="carousel">
+          <button class="button button--left">
+            <img class="button__image"
+              src="https://cdn.iconscout.com/icon/premium/png-512-thumb/left-caret-1578336-1341870.png">
+          </button>
+          <button class="button button--right">
+            <img class="button__image"
+              src="https://cdn.iconscout.com/icon/premium/png-512-thumb/left-caret-1578336-1341870.png">
+          </button>
+          <div class="card">
+            <div>
+              <h3 class="card__title">CUBA</h3>
+              <hr class="card__line-separator" />
+              <p class="card__description">Lorem Ipsum Dolor Sit Amet Consectetur Adipisicing Elit Sed Do Eiusmod Tempor
+                Incididunt Ut Labore Et Dolore Magna A</p>
+            </div>
+            <div>
+              <img class="card__image"
+                src="https://www.wendyperrin.com/wp-content/uploads/2016/04/vintage-cars-old-havana-cuba-cr-michael-petit.jpg"
+                alt="photo of cars in la havana Cuba." />
+            </div>
+          </div>
+          <div class="card">
+            <div>
+              <h3 class="card__title card__title--color-promo">New York</h3>
+              <hr class="card__line-separator" />
+              <p class="card__description">Lorem Ipsum Dolor Sit Amet Consectetur Adipisicing Elit Sed Do Eiusmo
+              </p>
+            </div>
+            <div>
+              <img class="card__image"
+                src="https://static.rentcars.com/imagens/modules/localidade/about/983-desktop-location-description.png"
+                alt="City of New York" />
+            </div>
+          </div>
+          <div class="card">
+            <div>
+              <h3 class="card__title">Paris</h3>
+              <hr class="card__line-separator" />
+              <p class="card__description">Lorem Ipsum Dolor Sit Amet Consectetur Adipisicing Elit Sed Do Eiusmod Tempor
+                Incididunt Ut Labore E
+                .</p>
+            </div>
+            <div>
+              <img class="card__image" src="https://turkishairlines.ssl.cdn.sdlmedia.com/637110514333635321TW.jpg"
+                alt="Citi of Paris France" />
+            </div>
+          </div>
+        </div>
+        <ul class="dots">
+          <li>
+            <button class="dot"></button>
+          </li>
```

Comment:
Maybe it is not necessary to use a list hear.
and do you think a `dot` or `dots container` could be a separate element of the carousel?

Reply:
Prefer it be a list due they contain more info.